src/SFA.DAS.FindEmployerSchemes.IntegrationTests/Web/Models/ErrorViewModelTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿
+using SFA.DAS.FindEmploymentSchemes.Web.Models;
+using Xunit;
+
+
+namespace SFA.DAS.FindEmploymentSchemes.IntegrationTests.Web.Models
+{
+    public class ErrorViewModelTests
```

Comment:
Could this be moved into the unit test project?

Reply:
This is already in the unit test project. Don't think we need it here also.

Reply:
As above, I've removed this too